.github/workflows/differential-shellcheck.yml

```diff
@@ -26,6 +26,6 @@ jobs:
           fetch-depth: 0
 
       - name: Differential ShellCheck
-        uses: redhat-plumbers-in-action/differential-shellcheck@60360c0fb283149ed03ad16b66257a967c3e5231
+        uses: redhat-plumbers-in-action/differential-shellcheck@v3
```

Comment:
Could you pin this to the specific SHA instead of a tag - `a14889568f6210b361eb29e16f3b07f512fca846` appears to be the SHA for `v3.0.1` (see https://github.com/systemd/systemd/pull/21316 for reasoning).